### ISSUE TITLE
Using %(RecursiveDir) with an extra slash throws

### ISSUE THREAD

**Issue by mattleibow** [mono/monodevelop#5490]
In my .csproj, I had this (note the slash between `RecursiveDir` and `Filename`):

```xml
<Compile Include="..\Code\**\*.cs" Link="%(RecursiveDir)\%(Filename)%(Extension)" />
```

This loads fine on Windows, but causes an exception on macOS. After some testing, I found that the path is being resolved incorrectly.

[Archive.zip](https://github.com/mono/monodevelop/files/2221172/Archive.zip)

> VS bug [#653157](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/653157)

**Comment by mattleibow** [comment]
After some debugging, it appears the that things go south here:
https://github.com/mono/monodevelop/blob/8aa00c4b8423485115e158b0766c918eca94de14/main/src/core/MonoDevelop.Core/MonoDevelop.Core/FileService.cs#L571

Without the extra slash: ".\MyFile.cs" => "MyFile.cs"
Wiith the extra slash: ".\\MyFile.cs" => "/MyFile.cs"

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.7.0.736 (master)

Author: Matthew Leibowitz
Commit: c0e65c30dbf4fc8698012f4c88863706960f4fd2 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.6.0.2100 (release-7.6)

Author: Matthew Leibowitz
Commit: 8d0a809284199f619fbb605497c959a247c2c338 (mono/monodevelop)
